### DISCUSSION TITLE
Popovers causing weird build errors

### DISCUSSION THREAD

**Discussion by macdonmc** [twilio-labs/paste#2336] [Q&A] [answered]
<img width="1501" alt="Screen Shot 2022-03-31 at 3 52 38 PM" src="https://user-images.githubusercontent.com/46056299/161163276-79a80c04-f0f7-45d6-bc99-bb3c7f4fecdc.png">

Hi there,

I'm using `PopoverContainer`, `PopoverButton`, and `Popover` from '@twilio-paste/core/popover', and my code is getting the errors above when i try to build it. I commented out the section of code where I use popovers and it builds perfectly fine (without the above error/any errors at all). Any ideas what might be causing this issue?

**Comment by andioneto** [comment]
Hi there @macdonmc 👋 

Could you please include some environment info so we can best help you? Please pass along the following as soon as possible:

Desktop information:
- OS: {{ your OS }}
- Version:

Dependency information:
- full Dependency list (with versions etc.)

**Reply by macdonmc** [reply]
OS 12.2.1 Monterey

Dependency list:

"dependencies": {
    "@twilio-paste/core": "^11.1.2",
    "@twilio-paste/popover": "^7.0.3",
    "@types/jest": "^27.4.1",
    "@types/node": "^17.0.21",
    "@types/react": "^17.0.40",
    "@types/react-dom": "^17.0.13",
    "prop-types": "^15.8.1",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router-dom": "^6.3.0",
    "typescript": "^4.5.5"
  },
  "devDependencies": {
    "@babel/core": "^7.17.5",
    "@storybook/addon-actions": "^6.4.19",
    "@storybook/addon-essentials": "^6.4.19",
    "@storybook/addon-interactions": "^6.4.19",
    "@storybook/addon-links": "^6.4.19",
    "@storybook/react": "^6.4.19",
    "@storybook/testing-library": "^0.0.9",
    "@testing-library/react": "^12.1.3",
    "@twilio-paste/core": "^11.1.2",
    "@twilio-paste/icons": "^6.2.1",
    "@twilio-paste/theme": "^6.0.3",
    "@twilio/flex-interactions-sdk": "^0.0.1",
    "@types/jest": "^27.4.1",
    "@types/node": "^17.0.21",
    "@types/react": "^17.0.40",
    "@types/react-dom": "^17.0.13",
    "@types/styled-system__theme-get": "^5.0.2",
    "babel-loader": "^8.2.3",
    "esbuild": "^0.14.27",
    "esbuild-node-externals": "^1.4.1",
    "prop-types": "^15.8.1",
    "react": "^17.0.2",
    "rimraf": "^3.0.2",
    "typescript": "^4.5.5"
  }

**Reply by macdonmc** [reply]
and i'm importing it from `core`: `import { PopoverContainer, PopoverButton, Popover } from '@twilio-paste/core/popover';`. tried importing it from `'@twilio-paste/popover'` as well and that produced the same error

**Comment by andioneto** [comment]
Hello @macdonmc, please let us know if you still need help with this!

**Comment by TheSisb** [comment, marked as answer]
Hi @macdonmc,

Thank you for reporting this issue. This seems to be a Typescript issue with one of our dependencies in newer versions of Typescript: https://github.com/ariakit/ariakit/issues/900

I'll work on a fix shortly.

**Reply by macdonmc** [reply]
Awesome thank you

**Reply by TheSisb** [reply]
A fix is being deployed right now and will be available in `@twilio-paste/core` version `11.5.0`. Please update and let us know if you have any more issues.

**Comment by samyzelly** [comment]


**Reply by SiTaggart** [reply]
@samyzelly I'm going to delete this reply because posting tokens and ids in a public forum is probably something you didn't mean to do. 

Hopefully you can read the email and give us details on your problem next time in a more secure way
